src/test/java/ContactListTest.java

```diff
@@ -211,12 +224,14 @@ void get_validIndex_returnsCorrectFriend() {
 
                 @Test
                 void get_negativeIndex_throwsIndexOutOfBoundsException() {
-                    assertThrows(IndexOutOfBoundsException.class, () -> classUnderTest.get(-1));
+                    Exception e = assertThrows(IndexOutOfBoundsException.class, () -> classUnderTest.get(-1));
+                    assertEquals("Cannot index with negative indices.", e.getMessage());
```

Comment:
Upon further thought, I am not sure the messages are a great thing to check. Exception, yeah, but message, idk. 

E.g., if the interface says some method should throw an exception, then yeah, check for the exception. But the interface doesn't say what the message should be. 

Am I crazy?

Reply:
checks out. 

Glad I cut them.